Resources/Prototypes/Entities/Markers/Spawners/Random/Food_Drinks/drinks_glass.yml

```
@@ -104,6 +104,7 @@
       - DrinkThreeMileIslandGlass
       - DrinkTortugaGlass
       - DrinkToxinsSpecialGlass
+      - DrinkScrantonSpecialGlass
```

Move this guy between `Sbiten` and `Screwdriver` above, please.

---

This also needs a moffstation guard
```suggestion
      - DrinkScrantonSpecialGlass # Moffstation
```